util/util.go

```diff
@@ -182,8 +225,18 @@ func Marshal(data interface{}) (v starlark.Value, err error) {
 			}
 		}
 		v = dict
+	case Marshaler:
+		v, err = x.MarshalStarlark()
 	default:
 		return starlark.None, fmt.Errorf("unrecognized type: %#v", x)
 	}
 	return
 }
+
+type Unmarshaler interface {
+	UnmarshalStarlark(starlark.Value) error
+}
+
+type Marshaler interface {
+	MarshalStarlark() (starlark.Value, error)
+}
```

Comment:
Same as `Unmarshaler`, this will cause `golint` to fail. Both the `Marshaler` interface and `MarshalStarlark` methods should have comments that explain their use.